Articles/Blog/lowndes_fellowship_post.md

```diff
@@ -0,0 +1,33 @@
+# How open source tooling is changing the way professional researchers learn to code
+
+<!---TODO: add Hero image--->
+
+The same open source tooling that is changing how we do science --- R, Python, Git, GitHub --- is changing the way we teach professional researchers how to code. Tooling like Jupyter Notebooks, RMarkdown, and now Quarto enable researchers to combine narrative, code, and outputs like figures in the same document, enabling transparency and reproducibility workflows that shared openly on GitHub. This same setup can be used to teach professional researchers to code, so that the teaching narrative unfolds interspersed with code the learner can run. But what is truly remarkable is the culture shift enabled by this setup --- learners have these documents in front of them that reinforces what the teacher screenshares live, and continues to be there for reference and self-paced learning when the teacher isn't there at all. This builds trust and resilience with learners, and role-models a new way of working openly --- all of this is a paradigm shift for how professional researchers learn new skills, work, and then teach others in turn. I am a marine scientist who can say from experience that this mode of teaching is game-changing; in fact I am making it my career to amplify this to empower more researchers. For my BSSw Fellowship, I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.
```

Comment:
```suggestion
The same open source tooling that is changing how we do science --- R, Python, Git, GitHub --- is changing the way we teach professional researchers how to code. Tooling like Jupyter Notebooks, RMarkdown, and now Quarto enable researchers to combine narrative, code, and outputs like figures in the same document, enabling transparency and reproducibility workflows that shared openly on GitHub. This same setup can be used to teach professional researchers to code, so that the teaching narrative unfolds interspersed with code the learner can run. Since the resources are openly available, and learners can work and learn in the same tools used to create those resources, there is consistency between what they see in the materials they learn from, what they see on their end, and the expected outcomes. But what is truly remarkable is the culture shift enabled by this setup --- learners have these documents in front of them that reinforces what the teacher screenshares live, and continues to be there for reference and self-paced learning when the teacher isn't there at all. This builds trust and resilience with learners, and role-models a new way of working openly --- all of this is a paradigm shift for how professional researchers learn new skills, work, and then teach others in turn. I am a marine scientist who can say from experience that this mode of teaching is game-changing; in fact I am making it my career to amplify this to empower more researchers. For my BSSw Fellowship, I modernized Openscapes' open educational resources that I first built with RMarkdown by refactoring them in Quarto, and have since used these updated resources to teach 6 cohorts (50 teams) of professional environmental and Earth scientists at NASA, NOAA, and the EPA.
```